extensions/amp-accordion/1.0/storybook/Basic.js

```diff
@@ -146,3 +151,35 @@ export const events = () => {
     </main>
   );
 };
+
+export const restructure = () => {
```

Comment:
I assume this is just a placeholder until the old storybook is rewritten?

Reply:
Correct, now that everything is merged, I've deleted and updated old storybooks.